packages/server-reason-react/server-reason-react.0.1.0/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+synopsis: "Rendering React components on the server natively"
+maintainer: ["David Sancho <dsnxmoreno@gmail.com>"]
+authors: ["David Sancho <dsnxmoreno@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/ml-in-barcelona/server-reason-react"
+bug-reports: "https://github.com/ml-in-barcelona/server-reason-react/issues"
+depends: [
+  "dune" {>= "3.9"}
+  "ocaml" {>= "5.0.0"}
+  "reason" {>= "3.10.0"}
+  "melange" {>= "3.0.0"}
+  "ppxlib" {> "0.23.0"}
+  "pcre" {<= "7.5.0"}
```

Comment:
```suggestion
  "pcre" {>= "7.4.0"}
```

Reply:
This fixes
```
#=== ERROR while compiling server-reason-react.0.1.0 ==========================#
# context              2.2.0~beta2~dev | linux/x86_64 | ocaml-base-compiler.5.1.1 | pinned(https://github.com/ml-in-barcelona/server-reason-react/releases/download/0.1.0/server-reason-react-0.1.0.tbz)
# path                 ~/.opam/5.1/.opam-switch/build/server-reason-react.0.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p server-reason-react -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/server-reason-react-7-ea8ede.env
# output-file          ~/.opam/log/server-reason-react-7-ea8ede.out
### output ###
# (cd _build/default && /home/opam/.opam/5.1/bin/ocamlopt.opt -w -40 -w -49 -g -I packages/melange.js/.js.objs/byte -I packages/melange.js/.js.objs/native -I /home/opam/.opam/5.1/lib/lwt -I /home/opam/.opam/5.1/lib/pcre -intf-suffix .ml -no-alias-deps -o packages/melange.js/.js.objs/native/js.cmx -c -impl packages/melange.js/Js.pp.ml)
# File "packages/melange.js/Js.ml", line 526, characters 17-30:
# 526 |     | Pcre.Error WorkspaceSize -> raise @@ Invalid_argument "WorkspaceSize"
#                        ^^^^^^^^^^^^^
# Error: This variant pattern is expected to have type Pcre.error
#        There is no constructor WorkspaceSize within type Pcre.error
# (cd _build/default && /home/opam/.opam/5.1/bin/ocamlc.opt -w -40 -w -49 -g -bin-annot -I packages/melange.js/.js.objs/byte -I /home/opam/.opam/5.1/lib/lwt -I /home/opam/.opam/5.1/lib/pcre -intf-suffix .ml -no-alias-deps -o packages/melange.js/.js.objs/byte/js.cmo -c -impl packages/melange.js/Js.pp.ml)
# File "packages/melange.js/Js.ml", line 526, characters 17-30:
# 526 |     | Pcre.Error WorkspaceSize -> raise @@ Invalid_argument "WorkspaceSize"
#                        ^^^^^^^^^^^^^
# Error: This variant pattern is expected to have type Pcre.error
#        There is no constructor WorkspaceSize within type Pcre.error
```

The upper bound is not necessary. If a new version of PCRE breaks your library we will add the upper bound at that stage when the revdeps are tested